iphone/Classes/TiUIListItem.m

```diff
@@ -420,9 +420,11 @@ - (void)setDataItem:(NSDictionary *)dataItem
                       }];
       }
     }
+    break;
```

Comment:
break should not be here. Next Case is expected to execute after above cases.

Reply:
Sorry for that, I thought it helps as well! But does the other change makes sense? Happy to understand this better

Reply:
@vijaysingh-axway Hans' earlier comment notes that importance of the braces - they force a new scope

Reply:
Okey. We can have braces. Only thing is we have to remove the above 'break' .  Because next Case is expected to execute after above case.